pkg/features/kube_features.go

```diff
@@ -793,6 +793,13 @@ const (
 	// https://github.com/kubernetes/kubernetes/issues/111516
 	SecurityContextDeny featuregate.Feature = "SecurityContextDeny"
 
+	// owner: @atosatto @yuanchen8911
+	// kep: http://kep.k8s.io/3902
+	// beta: v1.29
+	//
+	// Decouples taint-based Pod eviction from Node Lifecycle Controller.
+	SeparateTaintManager featuregate.Feature = "SeparateTaintManager"
```

Comment:
Everywhere we're using pretty consistently TaintManager, even the old packages, the KEP and everything else was using that. Yet somehow this PR decides to go from that to TaintEviction in package and file names? Why? What did I miss? I'd stick with the old name for consistency or rename the flag to reflect reality, but I remember there were some questions wrt renaming events, which that would also help solving.

Reply:
/cc @atosatto